extensions/common/context.h

```diff
@@ -54,6 +54,11 @@ const std::string kProtocolHTTP = "http";
 const std::string kProtocolGRPC = "grpc";
 const std::string kProtocolTCP = "tcp";
 
+const char kCanonicalServiceLabelName[] = "service.istio.io/canonical-name";
+const char kCanonicalServiceRevisionLabelName[] =
+    "service.istio.io/canonical-revision";
+const std::string kLatest = "latest";
```

Comment:
Can you please adhere to the modern standard (see ToW #140):
```c++
inline constexpr absl::string_view kLatest = "latest";
```

The problem with this code is that it duplicates variables every time this header is included.

Reply:
For protos, you can then use `proto.set_field(kLatest.data()` or `proto.set_field(kLatest.data(), kLatest.size())`

Reply:
string_view didn't work with the flatbuffers api. trial by experimentation led to `const char[]`.

Reply:
but i can update `kLatest`

Reply:
Just use `kLatest.data()`. Eventually it'll take string_view directly.

Reply:
string_view is slightly faster to convert to a string since it avoids the scan for `\0`.

Reply:
btw, for anyone looking for the absl tip of the week link, here is the one I think was meant: https://abseil.io/tips/168

Reply:
adding `inline` resulted in failures. are we 100% on c++17 (because my build process isn't)?

Reply:
You can drop inline but use constexpr. The build is probably C++11/14.

Reply:
should be done now.